# These topics are imported at launch time and cannot be changed during runtime

# All real vehicle related topics go here

# Actuators this node publishes to (real vehicle)
accelerator: /vehicle/throttle_cmd
brakes: /vehicle/brake_cmd
steering: /vehicle/steering_cmd
enable: /vehicle/enable
disable: /vehicle/disable
gear: /vehicle/gear_cmd
ulc: /vehicle/ulc_cmd

# Sensors this node subscribes to (published by vehicle)
report_accelerator: /vehicle/throttle_report
report_brakes: /vehicle/brake_report
report_steering: /vehicle/steering_report
report_gear: /vehicle/gear_report
enabled: /vehicle/dbw_enabled

# All vehicle control related topics go here (publish messages here to control ACTor)
drive: /actor/cmd_vel

# Status messages this node publishes - usually no changes needed here
status: /actor/status

# E-Stop related topics go here
estop_state: /actor/estop/state
estop_heartbeat: /actor/estop/heartbeat_edge
estop_physical_button: /actor/estop/physical_button
estop_wireless_button: /actor/estop/wireless_button
estop_software_button: /actor/estop/software_button
estop_trigger: /actor/estop/trigger
estop_reset: /actor/estop/reset
